File: legacy/check_laa.py
import struct

def is_laa(path):
    with open(path, 'rb') as f:
        f.seek(0x3C)
        pe_offset = struct.unpack('<I', f.read(4))[0]
        f.seek(pe_offset + 22)
        characteristics = struct.unpack('<H', f.read(2))[0]
        return (characteristics & 0x0020) == 0x0020

print("BAK LAA:", is_laa(r"D:\SteamLibrary\steamapps\common\Axiom Verge 2\AxiomVerge2.exe.bak"))
print("EXE LAA:", is_laa(r"D:\SteamLibrary\steamapps\common\Axiom Verge 2\AxiomVerge2.exe"))

File: legacy/check_laa.cs
using System;
using System.IO;

class Program {
    static void Main() {
        Console.WriteLine("BAK LAA: " + IsLAA(@"D:\SteamLibrary\steamapps\common\Axiom Verge 2\AxiomVerge2.exe.bak"));
        Console.WriteLine("EXE LAA: " + IsLAA(@"D:\SteamLibrary\steamapps\common\Axiom Verge 2\AxiomVerge2.exe"));
    }
    static bool IsLAA(string p) {
        using var fs = File.OpenRead(p);
        using var br = new BinaryReader(fs);
        fs.Seek(0x3C, SeekOrigin.Begin);
        int pe = br.ReadInt32();
        fs.Seek(pe + 22, SeekOrigin.Begin);
        return (br.ReadUInt16() & 0x0020) == 0x0020;
    }
}
